shared/src/commonMain/kotlin/com/example/tasky/repository/LoginRepository.kt

```diff
@@ -0,0 +1,14 @@
+package com.example.tasky.repository
+
+import com.example.tasky.dataSource.LoginDataSource
+import com.example.tasky.model.login.LoginBody
+import kotlinx.coroutines.flow.flow
+
+class LoginRepository(
+    private val loginDataSource: LoginDataSource = LoginDataSource(),
+) {
+    fun login(loginBody: LoginBody) =
+        flow {
+            emit(loginDataSource.login(loginBody))
+        }
```

Comment:
Is there a reason you're using a Flow here for a single API call? Generally, if you only expect one result from an async call, a suspend function is what should be used.

Reply:
oh, it is because it is easier to handle flow than suspend function in Swift. Let me fix it